src/Reflection/ReflectionClass.php

```diff
@@ -338,20 +338,33 @@ private function getMethodsIndexedByName() : array
     /**
      * Fetch an array of all methods for this class.
      *
+     * @param ?int $filter
```

Comment:
More documentation is needed on `$filter`, with examples, since this is a bitmask.